components/Dialogs/src/MDCAlertController.h

```diff
@@ -92,6 +92,7 @@
 
  Default value is NO.
  */
+// TODO : Deprecate in favor of MDCAlertControllerView
```

Comment:
can we add a `__deprecated_msg`? any reason why this is a TODO?

Reply:
Gone.